test/contracts/token/arcxRedemption.test.ts

```diff
@@ -0,0 +1,101 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address';
+import { TestToken, TestTokenFactory } from '@src/typings';
+import { ArcxRedemption } from '@src/typings/ArcxRedemption';
+import { MockArcxRedemptionFactory } from '@src/typings/MockArcxRedemptionFactory';
+import { addSnapshotBeforeRestoreAfterEach } from '@test/helpers/testingUtils';
+import { expect } from 'chai';
+import { BigNumber, utils } from 'ethers';
+import { ethers } from 'hardhat';
+
+const ARCX_AMOUNT = BigNumber.from(10).pow(18).mul(100)
```

Comment:
Can use ethers' utils, it's easier:
```suggestion
const ARCX_AMOUNT = utils.parseEther('100')
```